packages/roosterjs-editor-api/lib/format/clearFormat.ts

```diff
@@ -89,6 +89,8 @@ function clearAttribute(element: HTMLElement) {
     for (let attr of toArray(element.attributes)) {
         if (isTableCell && attr.name == 'style') {
             removeNonBorderStyles(element);
+        } else if (safeInstanceOf(element, 'HTMLTableElement') && attr.name == 'style') {

```

Comment:
Like line 87, you can move safeInstanceOf check out of the for loop so you only need to check it once